<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="assign_task_button">
      <property name="text">
       <string>Assign Task</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="create_button">
      <property name="text">
       <string>Create Task</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QToolButton" name="refresh_button">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>263</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
